src/engine/source/hlp2/src/alphanumeric.cpp

```diff
@@ -0,0 +1,55 @@
+#include <optional>
+#include <stdexcept>
+#include <string>
+#include <string_view>
+#include <vector>
+
+#include <fmt/format.h>
+
+#include <hlp/base.hpp>
+#include <hlp/hlp.hpp>
+#include <hlp/parsec.hpp>
+#include <json/json.hpp>
+#include <utils/stringUtils.hpp>
+#include <iostream>
+namespace hlp
+{
+parsec::Parser<json::Json> getAlphanumericParser(std::string name, Stop, Options lst)
```

Comment:
can we change name to a `const std::string &` ?

Reply:
Actually the same will go for the rest of the parsers. 
As stated in the coding style:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#const-and-exception-correctness